.gitignore

```diff
@@ -8,6 +8,7 @@
 result
 
 # nixago: ignore-linked-files
+.justfile
```

Comment:
I would typically not link it. The "on-ramp" can only work if that familiar file is there also when the user looks at Github.

That's actually the only (non-technical) reason to copy it, instead.